ws-nextjs-app/pages/[service]/send/[id]/Uploading/index.tsx

```diff
@@ -0,0 +1,16 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import Heading from '#app/components/Heading';
+import Paragraph from '#app/components/Paragraph';
+import styles from './styles';
+
+export default function Uploading() {
```

Comment:
Can this be a fat arrow function (we don't tend to use `function` in Simorgh)?

Reply:
Yea we have a bit of a mixture in the Next.js app. The Next docs tend to use `export default function` for components, but can change this.

Reply:
Ach I think we can leave it.

Reply:
No worries, let's leave it as is 👍